pkgs/development/python-modules/quamash/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, pytest, pyqt5, pyqt ? pyqt5 }:
+buildPythonPackage rec {
+  pname = "Quamash";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
    pname = "Quamash";
    inherit version;
```

I think the tarball is named that.